src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -1412,20 +1414,54 @@ sdap_nested_group_single_send(TALLOC_CTX *mem_ctx,
     return req;
 }
 
+static bool must_ignore(struct sdap_nested_group_single_state *state)
```

Comment:
I would suggest to not use the request specific `state` as argument but the DN string you want to check and the list of ignore DNs. This way it would be easier to re-use this function.

Reply:
Done